src/react/Link/Link.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ComponentPropsWithoutRef, JSX, ReactNode } from 'react'
```

Comment:
Assume this file meant to be internal – should we put internal things in an `internal` subdirectory?